Inc/ST-LIB_LOW/Communication/Server/Server.hpp

```diff
@@ -19,7 +23,8 @@ class Server{
 	};
 
 	ServerSocket *open_connection;
-	vector<ServerSocket*> running_connections;
+	array<ServerSocket*,10> running_connections;
```

Comment:
Two things here, you forgot to actually use the define MAX_CONNECTIONS_TCP_SERVER :) and instead of using a macro consider using constexpr, this way you have variable representing the length

Reply:
done